package.json

```diff
@@ -27,12 +27,26 @@
     "url": "https://github.com/lob/hapi-bookshelf-total-count/issues"
   },
   "homepage": "https://github.com/lob/hapi-bookshelf-total-count#readme",
+  "peerDependencies": {
+    "bookshelf": "0.9.x",
+    "hapi": "11.x"
+  },
```

Comment:
Ah I meant to use `"bookshelf": "0.x"` not `"bookshelf": "0.9.x"`. That would resolve what you're pointing out right?